style_editor.html

```diff
@@ -1149,8 +1162,10 @@
 function initGL() {
   AddTab("color", "Styles",effect_links.sort().join(""))
   AddTab("rgb", "Colors",
-    "<input type=color id=COLOR value='#ff0000' onclick='ClickColor()' />" +
-    qlinks.sort().join(""));
+    qlinks.sort().join("") +
+    "<div class='custom-color'>" +
+    "<label>Custom color </label>" +
```

Comment:
Could you please use the "for" attribute to tie the label to the input?

Reply:
good call, thanks.